app/services/event.service.js

```diff
@@ -5,12 +5,9 @@
     .module('calendar')
     .factory('eventService', eventService);
 
-  eventService.$inject = ['$http', '$window', '$q', 'utilityService', 'taxonomyService', 'siteService', 'dateService', '$route', '$location'];
+  eventService.$inject = ['$http', '$window', '$q', 'utilityService', 'taxonomyService', 'siteService', 'dateService'];
 
-  function eventService($http, $window, $q, utilityService, taxonomyService, siteService, dateService, $route, $location) {
-
-    //Check for external hooks, or set null.
-    window.UniCal = window.UniCal || [];
```

Comment:
What is the reasoning behind removing this? I'm pretty sure we need this here in order to support external hooks.